opencensus/exporters/stats/stackdriver/stackdriver_exporter.h

```diff
@@ -36,6 +36,12 @@ struct StackdriverOptions {
 
   // The RPC deadline to use when exporting to Stackdriver.
   absl::Duration rpc_deadline = absl::Seconds(5);
+
+  // If empty, the exporter will use "custom.googleapis.com/opencensus"
+  std::string metric_domain;
```

Comment:
Please explain with probably an example of a metric how this will be used.

Reply:
Added docs that make it look like the Java version, and also renamed to `metric_name_prefix` to match Java.